pubspec.yaml

```diff
@@ -5,7 +5,7 @@ author: Steven Roose <stevenroose@gmail.com>
 homepage: https://github.com/stevenroose/dart-eventsource
 
 environment:
-  sdk: ">=1.0.0 <3.0.0"
+  sdk: ">=2.7.0 <3.0.0"
```

Comment:
So this breaks support for Dart 1.0? So we'll have to release this as a breaking semver bump.. Is it not possible to do this in a backwards compatible way?

Reply:
it wouldn't compile without it, something changed. :-(